src/main/java/Rolex.java

```diff
@@ -0,0 +1,140 @@
+    }
+
+    public static void inputIsBye(){
+        printLines();
+        System.out.println("Bye. Hope to see you again soon!\n");
+        printLines();
+    }
+
+    public static void inputIsList(){
+        if(taskCount>=1){
+            printLines();
+            int indexNum = 1;
+            System.out.println("Here are the tasks in your list:");
+            for(int i = 0; i<taskCount; i++){
+                System.out.println(indexNum + ". " + task[i]);
+                indexNum++;
+            }
+            printLines();
+        } else{
+            RolexException.detectError("list");
+        }
+    }
+
+    public static void inputIsMark(String userInput){
+        int index = Integer.parseInt(userInput.substring(5));
+        if(index>0 && index<=taskCount) {
+            task[index-1].MarkTask();
+        } else{
+            RolexException.detectError(userInput);
+        }
+    }
+
+    public static void inputIsUnmark(String userInput){
+        int index = Integer.parseInt(userInput.substring(7));
+        if(index>0 && index<=taskCount) {
+            task[index-1].unMarkTask();
+        } else{
+            RolexException.detectError(userInput);
+        }
+    }
+
+    public static void addPrintTask(){
+        printLines();
+        System.out.println("Got it. I've added this rolex.task:");
+        System.out.println(task[taskCount-1]);
+        System.out.println("\nNow you have " + taskCount + " tasks in the list.");
+        printLines();
+    }
+
+    public static void inputIsTodo(String userInput){
+        if(userInput.startsWith("todo")){
+            RolexException.detectError(userInput);
+        } else{
+            String todoName = userInput.substring(5);
+            task[taskCount] = new Todo(todoName);
+            taskCount++;
+            addPrintTask();
+        }
+    }
+
+    public static void inputIsDeadline(String userInput){
+        int indexOfBy = userInput.indexOf("/by");
+        if(indexOfBy==-1){
+            RolexException.detectError(userInput);
+        } else{
+            String deadlineName = userInput.substring(9,indexOfBy-1);
+            String by = userInput.substring(indexOfBy+3);
+            task[taskCount] = new Deadline(deadlineName, by);
+            taskCount++;
+            addPrintTask();
+        }
+    }
+
+    public static void inputIsEvent(String userInput){
+        int indexOfFrom = userInput.indexOf("/from");
+        int indexOfTo = userInput.indexOf("/to");
+        if(indexOfFrom == -1 || indexOfTo == -1){
+            RolexException.detectError(userInput);
+
+        } else{
+            String eventName = userInput.substring(6,indexOfFrom-1);
+            String startTime = userInput.substring(indexOfFrom+6,indexOfTo-1);
+            String endTime = userInput.substring(indexOfTo+4);
+            task[taskCount] = new Event(eventName, startTime, endTime);
+            taskCount++;
+            addPrintTask();
```

Comment:
Seems similar to 87-89 and  74-76, perhaps you could combine them in 1 method since apart from the type of task being instantiated, they exhibit similar behaviour